index.js

```diff
@@ -0,0 +1,43 @@
+const fs = require('fs');
+const { spawnSync } = require('child_process');
+const os = require('os');
+const readline = require('readline');
+const { getCurrentUnixTime, clearLine, getComand,logFileName, windowsCommand, unixLikeOSCommand, timeToRefresh } = require('./helpers');
+
+if (!fs.existsSync(logFileName)) {
+  fs.writeFileSync(logFileName, '');
+}
+
+const rl = readline.createInterface({
+    input: process.stdin,
+    output: process.stdout,
+  });
+
+function monitorProcesses() {
+  const currentPlatform = os.platform();
+  const command = getComand(currentPlatform);
+
+  setInterval(() => {
+    const currentTime = getCurrentUnixTime();
+    const result = spawnSync(command, { shell: true, encoding: 'utf-8' });
```

Comment:
You could you exec here which creates a shell automatically.

Reply:
Fixed, thaks.